WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -13,4 +13,38 @@ def competition_message_for_user(competition, user)
                 end
     messages.join(' ')
   end
+
+  def announced_class(distance_announced)
```

Comment:
Could you rename the variables in these three methods? I honestly don't know what unit "distance" is referring to here. Is it days?